src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -4,12 +4,16 @@ import io from 'socket.io-client'
 
 import theme from '../../Theme/style.css'
 import style from './style.css'
+import { performHttpReq } from '../../utils/http'
 import Spinner from '../../Spinner'
+import PhoneNumberInputLazy from '../../PhoneNumberInput/Lazy'
+import Error from '../../Error'
 import { trackComponent } from '../../../Tracker'
 
 class CrossDeviceLink extends Component {
   constructor(props) {
     super(props)
+    this.state = {mobileNumber: null}
```

Comment:
I think we can tidy up how this file uses the mobileNumber. Currently it's part of the state of two components and it's in the redux store, and has a couple of helper methods, but we only use it when sending to the SMS.

Also when you press 'resend link' we don't populate the text field even though we have the number saved.

Reply:
The repopulation of the field was not part of the acceptance criteria. We use the mobileNumber state to know the state of the validation field (whether it contains a valid number or not and show the number accordingly) and to pass it to the redux store so it can be displayed in the sms delivery success screen

Reply:
OK so keeping it in the store makes sense, but then we should be able to get rid of it from the state. I don't think we need it in the CrossDeviceLink component at all.

My view is that acceptance criteria are just some minimum features. Given how easy it would be to populate the number field my view is that we should just do it if @danielspagnolo thinks it's a good idea.

Reply:
I spoke to @danielspagnolo and he thinks it's a good idea so I changed to have a default value

Reply:
Great, this state is still unused